lib/iris/tests/unit/fileformats/netcdf/test_save.py

```diff
@@ -59,6 +59,22 @@ def test_attributes_arrays(self):
             ds.close()
         self.assertArrayEqual(res, np.arange(2))
 
+    def test_no_special_attribute_clash(self):
+        # Ensure that saving multiple cubes with netCDF4 protected attributes works as expected.
+        # Note that here we are testing variable attribute clashes only - by saving multiple 
+        # cubes the attributes are saved as variable attributes rather than glocal attributes.
```

Comment:
Typo: glocal -> global. The line length looks too long also, it should be <80 characters. Also some trailing whitespace that needs to go ([the Travis CI test result](https://travis-ci.org/SciTools/iris/jobs/134382366) for coding standards should list lines and columns for these).